tools/check_format.py

```diff
@@ -78,6 +78,118 @@
 LIBCXX_REPLACEMENTS = {
     "absl::make_unique<": "std::make_unique<",
 }
+
+UNOWNED_EXTENSIONS = {
+  "extensions/filters/http/ratelimit",
+  "extensions/filters/http/buffer",
+  "extensions/filters/http/grpc_http1_bridge",
+  "extensions/filters/http/grpc_http1_reverse_bridge",
+  "extensions/filters/http/rbac",
+  "extensions/filters/http/gzip",
+  "extensions/filters/http/ip_tagging",
+  "extensions/filters/http/tap",
+  "extensions/filters/http/fault",
+  "extensions/filters/http/grpc_json_transcoder",
+  "extensions/filters/http/health_check",
+  "extensions/filters/http/router",
+  "extensions/filters/http/cors",
+  "extensions/filters/http/ext_authz",
+  "extensions/filters/http/dynamo",
+  "extensions/filters/http/lua",
+  "extensions/filters/http/grpc_web",
+  "extensions/filters/http/common",
+  "extensions/filters/http/common/aws",
+  "extensions/filters/http/squash",
+  "extensions/filters/common",
```

Comment:
Can we maybe filter out directories with common in it? These are things that are shared by other extensions which should have owners?

Reply:
We totally could.  I think it's a bit nicer to list them explicitly so if we do code reviewer assignment we have explicit owners.  We could arguably script around it but I think it's easier to have it explicit.  WDYT?

Reply:
Sure, no strong feelings on this if you think this is better.

Reply:
Yeah, mild preference for this way, just for tooling.
Anything else you want to see here?